scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.js

```diff
@@ -66,7 +66,7 @@ export function ContentProfileSchemaEditor(content, config) {
 
             scope.formattingOptionsEditor3 = [
                 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'ordered list', 'unordered list', 'quote', 'media', 'link',
-                'embed', 'underline', 'italic', 'bold', 'table'
+                'embed', 'underline', 'italic', 'bold', 'table', 'paragraph mark'
```

Comment:
Since you also added support for displaying spaces, it might be good to rename 'paragraph mark' to 'invisibles' or something similar.